common/Linux/ScreenSaverDBus.cpp

```diff
@@ -0,0 +1,81 @@
+/*  PCSX2 - PS2 Emulator for PCs
```

Comment:
Given it's only a single function, you could just put it in `LnxMisc.cpp` (guarded by `#ifdef DBUS_API`). Save a tiny bit of compile time.